qiskit/optimization/algorithms/cplex_optimizer.py

```diff
@@ -18,8 +18,8 @@
 import logging
 from typing import Optional
 
-from qiskit.aqua import MissingOptionalLibraryError
```

Comment:
You cannot remove it.

Reply:
It was mistakenly removed(maybe due to merge conflict). I have added it back (will be done in next commits).